docker-compose.yml

```diff
@@ -1,51 +1,61 @@
-version: '3.9'
```

Comment:
I guess that the `version` property has been deprecated. Docker told me it does nothing and I should remove it.